Makefile

```diff
@@ -70,6 +70,7 @@ install-mk: n64.mk
 install: install-mk libdragon
 	install -Cv -m 0644 libdragon.a $(INSTALLDIR)/mips64-elf/lib/libdragon.a
 	install -Cv -m 0644 n64.ld $(INSTALLDIR)/mips64-elf/lib/n64.ld
+	install -Cv -m 0644 rsp.ld $(INSTALLDIR)/mips64-elf/lib/rsp.ld
```

Comment:
Probably this would have to be done for the RSP compilation rule:

```
$(BUILD_DIR)/%.o: $(SOURCE_DIR)/%.S $(N64_LIBDIR)/rsp.ld
```

but the problem is that `$(N64_LIBDIR)/rsp.ld` only exists after libdragon is installed, and that rule needs to work also for building libdragon itself (since libdragon needs ucode). This is actually similar to n64.mk issues where we ended up installing it before anything else. So if we wanted to go this route, I think we should also install `n64.ld` in the `install-mk` rule (maybe renaming it `install-pre` or something), which also requires sudo etc. So I guess it would require some testing so I'd rather postpone it.

Reply:
You are right, just opened an issue in order to not forget about it: #323